packages/core/src/editor/Block.css

```diff
@@ -114,13 +114,13 @@ NESTED BLOCKS
 }
 
 /* HEADINGS*/
-[data-level="1"] {
+[data-content-type="heading"] {
```

Comment:
because "data-level=1" is the default, it would not be part of the html output anymore